spec/interactions/add_dragon_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+
+def create_data
+  let!(:dragon_type) { FactoryBot.create :dragon_type }
```

Comment:
Wszysgtkie te `let()` powinny się znaleźć wewnątrz `RSpec.describe AddDragon do`

Powinny być też "luzem", nie opakowane w metodę `create_data`

Reply:
Trzeba je też podzielić według testów do których należą. np. 
`let!(:user) { FactoryBot.create :user }` jest dla wszystkich testów, więc wystarczy go zdefiniować raz tutaj:
```
RSpec.describe AddDragon do
  let!(:user) { FactoryBot.create :user }

  describe '#pay_for_dragon' do
```

niektóre powinny być wewnątrz `When User has to pay for dragon` (zgodnie z logiką testów)